<launch>

	<node name="pr2_gripper_feedback_binned" pkg="feedback_devices" type="pr2_gripper_feedback_binned" output="screen" />
	<node name="tacta_grippers" pkg="feedback_devices" type="tacta_grippers" output="screen" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find feedback_devices)/rviz/pr2_tacta_grippers.vcg" />
</launch>
